lib/Parser/DefinitionList.php

```diff
@@ -0,0 +1,24 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Doctrine\RST\Parser;
+
+class DefinitionList
+{
+    /** @var DefinitionListTerm[] */
+    private $terms = [];
+
+    public function addTerm(DefinitionListTerm $term) : void
```

Comment:
Would it be possible to have a more complex constructor, instead of defining the terms as a mutable growing stack?

Reply:
Yes, re your other comment, we can build all the data up as an array first then pass everything through the constructor.